platform-server-rest/src/main/java/uno/cod/platform/server/rest/controller/TaskController.java

```diff
@@ -22,18 +24,21 @@ public TaskController(TaskService taskService) {
     }
 
     @RequestMapping(value = RestUrls.TASKS, method = RequestMethod.POST)
+    @PreAuthorize("isAuthenticated() and @securityService.isOrganizationAdmin(principal, #dto.organizationId)")
     public ResponseEntity<String> create(@Valid @RequestBody TaskCreateDto dto) {
         taskService.save(dto);
         return new ResponseEntity<>(HttpStatus.CREATED);
     }
 
     @RequestMapping(value = RestUrls.TASKS_ID, method = RequestMethod.GET)
+    @PreAuthorize("isAuthenticated()")
```

Comment:
tasks should only be displayed if they're invited to a challenge that contains the task